src/pages/index.tsx

```diff
@@ -1,3 +1,62 @@
+import { Post } from "@/shared/components/index";
+import { useEffect, useState } from "react";
+import styles from "@/pages/index.module.scss";
+import instance from "./api/axiosInstance";
+
+type Res = {
+  item: {
+    id: string;
+    hourlyPay: number;
+    startsAt: string;
+    workhour: number;
+    description: string;
+    closed: boolean;
+    shop: {
+      item: {
+        id: string;
+        name: string;
+        category: string;
+        address1: string;
+        address2: string;
+        description: string;
+        imageUrl: string;
+        originalHourlyPay: number;
+      };
+    };
+  };
+};
+
+type Props = {
+  item: Res[];
+};
+
 export default function Home() {
-  return <div>메인페이지</div>;
+  const [userData, setUserData] = useState<Props["item"]>([]);
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await instance.get(`notices`);
```

Comment:
문자열만 사용할때는 백틱보다는 작은따옴표나 큰따옴표로 표현해주시면 좋을 것 같습니다~
```suggestion
      const response = await instance.get('notices');
```